packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/platform_interface/google_maps_inspector_platform.dart

```diff
@@ -137,4 +137,13 @@ abstract class GoogleMapsInspectorPlatform extends PlatformInterface {
       {required int mapId, required ClusterManagerId clusterManagerId}) {
     throw UnimplementedError('getClusters() has not been implemented.');
   }
+
+  /// Returns current camera position.
```

Comment:
@ditman should I add following method here and use it in tests to see if getting camera position is supported on the platform.
Then I can drop the changes for web platform from this PR and everything on it's separate PR later after problems with it are solved?

```dart
  /// If the platform supports getting camera position
  bool supportsGettingGameraPosition() => false;
```

Reply:
That sounds good; pretty much any time we can isolate platform-specific limitations behind a generic support query it makes the rest of the code more maintainable.

Reply:
Done at https://github.com/flutter/packages/pull/7648/commits/f4a39ce081257bcbfe2d38e9ffa637023c2f318e